octobot/community/supabase_backend/community_supabase_client.py

```diff
@@ -263,10 +264,27 @@ async def fetch_bot_profile_data(self, bot_config_id: str) -> commons_profiles.P
             ].get("minimal_funds", [])
         ] if bot_config[enums.BotConfigKeys.EXCHANGES.value] else []
         profile_data.profile_details.version = bot_config["product_config"][enums.ProfileConfigKeys.VERSION.value]
+        profile_data.trader_simulator.enabled = bot_config.get("simulated", False)
+        if profile_data.trader_simulator.enabled:
+            portfolio = (bot_config.get(
+                enums.BotConfigKeys.OPTIONS.value
+            ) or {}).get("portfolio")
+            usd_like_asset = "USDT"  # todo use dynamic value when exchange is not supporting USDT
```

Comment:
Could we use the strategy reference market if it's an usd like asset?

Reply:
good idea 👍